src/scss/includes/components/button.scss

```diff
@@ -14,6 +14,12 @@
     color: $action-blue-base;
   }
 
+  &--shadow {
```

Comment:
Do you think we could set a shadow independently of the variant ?
I mean, this props could be composed : we could have a variant with, or without a shadow props maybe ?

Reply:
It makes sense, but here this would require defining first a variant without a border, then having a prop and CSS class for the shadow. If you think it's worth it I'm ok with it.